### ISSUE TITLE
AVT Issue - The tabbable element's role 'none' is not a widget role

### ISSUE THREAD

**Issue by AmirHussain93** [IBM/carbon-components-angular#2296]
## Detailed description
`<ibm-row-list>` is having accessibility violation. 
The tabbable element's role 'none' is not a widget role

> Is this a feature request (new component, new icon), a bug, or a general issue?
bug

> Is this issue related to a specific component?
yes, `<ibm-structured-list>`

> What did you expect to happen? What happened instead? What would you like to see changed?
A tabbable element must have a valid widget role

> What browser are you working in?
Chrome

> What version of the Carbon Design System are you using?
"carbon-components": "^10.56.0",

## Additional information

![image](https://user-images.githubusercontent.com/27803197/196399098-a96d3115-1d9f-4c83-a5fc-9029e05273e6.png)

**Comment by Akshat55** [comment]
This issue has been addressed in v11 (next) via PR #2287 - should be back-ported.
